### ISSUE TITLE
Please add the fact that WSL2 does not run systemd (or anything that systemd usually starts, like dbus-daemon)

### ISSUE THREAD

**Issue by MedAnd** [MicrosoftDocs/WSL#434]
As per feedback from @therealkenc on issue "WSL2: Service Fabric for Linux Installation Errors" - https://github.com/microsoft/WSL/issues/4178

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 57e78ef3-d2eb-1708-d319-be5c4d72274b
* Version Independent ID: 3f3f241f-6e72-859d-3d93-e115e2de0bfb
* Content: [WSL 2 Frequently Asked Questions](https://docs.microsoft.com/en-us/windows/wsl/wsl2-faq#feedback)
* Content Source: [WSL/wsl2-faq.md](https://github.com/MicrosoftDocs/WSL/blob/live/WSL/wsl2-faq.md)
* Product: **windows-subsystem-for-linux**
* GitHub Login: @mscraigloewen
* Microsoft Alias: **mscraigloewen**

**Comment by therealkenc** [comment]
> Content Source: WSL/wsl2-faq.md

There is a feedback mechanism at the bottom of the [FAQ](https://docs.microsoft.com/en-us/windows/wsl/wsl2-faq) page. UserVoice for systemd is [here](https://wpdev.uservoice.com/forums/266908-command-prompt-console-windows-subsystem-for-l/suggestions/16571479-support-for-running-systemd) and (by association) [here](https://wpdev.uservoice.com/forums/266908-command-prompt-console-windows-subsystem-for-l/suggestions/19243282-run-services-when-windows-starts). Discussion thread for systemd is dupe #994.

[fwiw, the **q**uestion (let's say) "Do Linux distributions from the Store launch `systemd` on WSL?" isn't actually **a**sked very **f**requently.]

**Comment by MedAnd** [comment]
Re: systemd, I should clarify that I'm not opinionated about how software runs on WSL2 as long as it installs, runs and is supported one day. 

So far two important MS packages I tested do not even install... Service Fabric for Linux and the Azure Monitor for Linux (aka OMS for Linux agent). I assume SQL Server 2019 Preview for Linux does not work either.... if the issue is the same in all cases why not unblock this scenario, why not get existing MS software running on WSL2?

As I'm not sure how else to engage with the team hence this feedback, but my point is if WSL2 is going to be useful & production grade one day, there must be guidance on getting packages such as Service Fabric, OMS Agent, SQL Server etc working and supported?
 
For example create WSL2 compliance guidelines, how to port existing solutions (explaining how not to use systemd and instead use approach x), would love to see a blog post from @mscraigloewen @benhillis talking about this vision and road-map 😊

PS. The potential of WSL2 is evident... I just want my favorite MS Linux software to run on it!

**Comment by gsilvestri** [comment]
Why in the hell You are using MS Windows sofware ported to Linux in a virtual environment inside a Windows 10 installation in the first place??

Do You know that the more you port the more problems can arise??

**Comment by cerebrate** [comment]
Although if that's going to be added, it's probably also worth mentioning that while it may not run systemd by default, it's Not Hard to make it run systemd. 😊

**Comment by MedAnd** [comment]
Best for WSL2 team to comment but if it’s simple, wondering why not support this through wsl config? 😊

**Comment by khuongduybui** [comment]
You may check out https://github.com/arkane-systems/genie

**Comment by jovton** [comment]
The following post is about enabling snap, but since is needs systemd, it is quite informative. It is also very short: https://forum.snapcraft.io/t/running-snaps-on-wsl2-insiders-only-for-now/13033. Works really well.

**Comment by NotTheDr01ds** [comment]
Resolved by #1619, it seems to me.  Recommend closing.

**Comment by MedAnd** [comment]
@NotTheDr01ds - is this feature available to users of Windows 10 WSL2?

**Comment by NotTheDr01ds** [comment]
@MedAnd [Yes](https://devblogs.microsoft.com/commandline/the-windows-subsystem-for-linux-in-the-microsoft-store-is-now-generally-available-on-windows-10-and-11/)! ;-)

**Comment by MedAnd** [comment]
@NotTheDr01ds - my understanding was this was only supported in the latest version of WSL2, distributed via the MS Store and for Windows 11, not available on Windows 10 WSL2... would you have a link to the release notes confirming?

**Comment by NotTheDr01ds** [comment]
@MedAnd Apologies if you missed the link I provided above on my "Yes" - I know it was quite short. ;-).

But the title of the Blog post (from the WSL Program Manager) that I linked in the [Yes](https://devblogs.microsoft.com/commandline/the-windows-subsystem-for-linux-in-the-microsoft-store-is-now-generally-available-on-windows-10-and-11/), is *"The Windows Subsystem for Linux in the Microsoft Store is now generally available on Windows 10 and 11"*.

And to quote from that post:

> WSL in the Store will now also be available on Windows 10 in addition to Windows 11. So, Windows 10 users will also be able to enjoy all of the latest features for WSL **including systemd** and Linux GUI app support!

(emphasis added)

And I can personally confirm that I have a Windows 10 system here running 1.0.3 with Ubuntu 22.04 and Systemd enabled and running.

As for release notes, oddly, [they](https://github.com/microsoft/WSL/releases/tag/1.0.0) skipped the biggest feature (Windows 10 support).  This is why, when it came out, I myself didn't notice that Windows 10 was supported for a few days.  I check the "releases" here more often than I check the DevBlog posts ;-).

But yes, Systemd support is available on Windows 10 in 1.0.0 and later.  Your 1.0.3.0 release will support it on Windows 10 just the same as on Windows 11.
